schema/spacewalk/common/tables/rhnServerPath.sql

```diff
@@ -27,13 +27,16 @@ CREATE TABLE rhnServerPath
     created          TIMESTAMPTZ
                          DEFAULT (current_timestamp) NOT NULL,
     modified         TIMESTAMPTZ
-                         DEFAULT (current_timestamp) NOT NULL
+                         DEFAULT (current_timestamp) NOT NULL,
+
+    CONSTRAINT rhn_serverpath_sid_pos UNIQUE
```

Comment:
Unique index can´t be used with DEFERRABLE INITIALLY DEFERRED.
Hibernate transaction first adds new entries then deletes the old ones and during this the constraint is not satisfied.
The old code never hit the problem because the path was only created and never modified.